### PR TITLE
feat: add API reference for Tenant Members and Tenant Invitations

### PR DESCRIPTION
**Pull request by koalaty-code**
## Description

- Adds API Reference documentation for user management operations
  - Create Tenant Invitation
  - Resend Tenant Invitation
  - List Tenant Invitations
  - Get Tenant Invitation By ID
  - Delete Tenant Invitation
  - List Tenant Members
  - Delete Tenant Member

## Testing required outside of automated testing?

- [X] Not Applicable

### Screenshots (if appropriate):

- [X] Not Applicable

## Rollback / Rollforward Procedure

- [X] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: source/includes/api-reference/_tenant_members.md [hunk 1]
```diff
@@ -0,0 +1,514 @@
...
+
+## Tenant Member Object
+
+| Attribute     | Type                                         | Description                                                                                        |
+|---------------|----------------------------------------------|----------------------------------------------------------------------------------------------------|
+| `id`          | *uuid*                                       | Unique identifier of the Tenant Member                                                             |
+| `tenant_id`   | *uuid*                                       | The [Tenant](#tenants-tenant-object) ID that the membership is attached                            |
+| `role`        | *string*                                     | The role assigned to the Tenant Member                                                             |
+| `user`        | *[user object](#tenant-members-user-object)* | The User which this membership is attached to                                                      |
+| `created_by`  | *uuid*                                       | (Optional) The ID of the User that created the Tenant Member                                       |
```
Comment (bweber) [inline source/includes/api-reference/_tenant_members.md:15]: These would always be populated, right?

File: source/includes/api-reference/_tenant_members.md [hunk 2]
```diff
@@ -0,0 +1,514 @@
...
+| `created_at`  | *date*                                       | (Optional) Created date of the Tenant Member in ISO 8601 format                                    |
+| `modified_by` | *uuid*                                       | (Optional) The ID of the User or [Application](#applications) that last modified the Tenant Member |
+| `modified_at` | *date*                                       | (Optional) Last modified date of the Tenant Member in ISO 8601 format                              |
+
+## User Object
+
+| Attribute    | Type     | Description                                     |
+|--------------|----------|-------------------------------------------------|
+| `id`         | *uuid*   | Unique identifier of the User                   |
+| `email`      | *string* | (Optional) The e-mail address of the User       |
```
Comment (bweber) [inline source/includes/api-reference/_tenant_members.md:25]: What scenario would we not have an email?

File: source/includes/api-reference/_tenant_members.md [hunk 3]
```diff
@@ -0,0 +1,514 @@
...
+| `status`      | *string* | The status of the invitation (i.e. `PENDING` or `EXPIRED`)                                                                                                                                                                                                             |
+| `expires_at`  | *date*   | The expiration date of the invitation is ISO 8601 format. By default, the invitation expires 72 hours from the time it was created. Invitations can be resent to extend the expiration, but doing so will invalidate the previous invitation link sent to the invitee. |
+| `created_by`  | *uuid*   | (Optional) The ID of the User that created the Tenant Invitation                                                                                                                                                                                                       |
+| `created_at`  | *date*   | (Optional) Created date of the Tenant Invitation in ISO 8601 format                                                                                                                                                                                                    |
```
Comment (bweber) [inline source/includes/api-reference/_tenant_members.md:40]: These would always be populated, right?

File: source/includes/api-reference/_tenant_members.md (hunk 1 above)
Comment (koalaty-code) [reply source/includes/api-reference/_tenant_members.md:15]: Yes, but for consistency we expose them as optional across all audited models.

File: source/includes/api-reference/_tenant_members.md (hunk 2 above)
Comment (koalaty-code) [reply source/includes/api-reference/_tenant_members.md:25]: If the user's account gets deleted, all we retain is the user's ID.

**Review by bweber — APPROVED** [review]

**Pull request merged**